dynamic-demo-plugin/src/components/ClusterOverview/Inventory.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { MonitoringIcon } from '@patternfly/react-icons';
+import {
+  K8sResourceCommon,
+  useK8sWatchResource,
+  InventoryItem,
+  InventoryTitle,
+  InventoryBody,
+  InventoryStatus,
+  InventoryLoading,
+} from '@openshift-console/dynamic-plugin-sdk';
+
+const WorkerNodeInventory: React.FC<{}> = () => {
```

Comment:
```suggestion
const WorkerNodeInventory: React.FC = () => {
```

Also note that

> The **_type_** `{}` doesn't mean "any empty object", it means "any non-nullish value".

see [here](https://github.com/typescript-eslint/typescript-eslint/issues/2063#issuecomment-675156492) for details.

Reply:
ooh, good to know! Fixed